scikeras/wrappers.py

```diff
@@ -345,27 +346,36 @@ def _get_compile_kwargs(self):
         compile_kwargs = route_params(
             init_params, destination="compile", pass_filter=self._compile_kwargs,
         )
-        compile_kwargs["optimizer"] = _class_from_strings(
-            compile_kwargs["optimizer"], optimizers_module.get
-        )
+        try:
+            compile_kwargs["optimizer"] = _class_from_strings(
+                compile_kwargs["optimizer"], optimizers_module.get
+            )
+        except ValueError:
+            pass  # unknown optimizer
         compile_kwargs["optimizer"] = unflatten_params(
             items=compile_kwargs["optimizer"],
             params=route_params(
                 init_params, destination="optimizer", pass_filter=set(), strict=True,
             ),
         )
-        compile_kwargs["loss"] = _class_from_strings(
-            compile_kwargs["loss"], losses_module.get
-        )
+        try:
+            compile_kwargs["loss"] = _class_from_strings(
+                compile_kwargs["loss"], losses_module.get
+            )
+        except ValueError:
+            pass  # unknown loss
```

Comment:
really only this one is needed, to pass through "auto". But this defense is probably a good idea on all of them.

Reply:
Removed via https://github.com/adriangb/scikeras/pull/210/files#r587758575